agent/unit-test/configstore_unittest.cpp

```diff
@@ -96,12 +85,12 @@ TEST_F(ConfigStoreTest, constructorDir) {
 /* Constructing a FilteredConfigStore with a directory path,
  * even if all files can not be parsed, should not throw an exception.
  */
-/* This test fails intermittently, not clear why
-TEST_F(FilteredConfigStoreTest, constructorDir2) {
+//This test fails intermittently, not clear why
+TEST_F(ConfigStoreTest, constructorDir2) {
     // No config files, but ok dir
     ASSERT_NO_THROW(FilteredConfigStore("/home/vagrant/softwarecontainer/"));
```

Comment:
How come an absolute path works here? If it's not actually used, perhaps it's better to change it to something more obviously irrelevant. If we rely on it, it will fail if run in another environment.